packages/vm/src/modules/sized_module.rs

```diff
@@ -1,9 +1,11 @@
-use wasmer::{Module, Store};
+use crate::Size;
+use wasmer::{Engine, Module};
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
 pub struct CachedModule {
-    pub store: Store,
+    pub engine: Engine,
     pub module: Module,
+    pub store_memory_limit: Option<Size>,
```

Comment:
Do we need to cache this? Can't we just use a new value from `struct CacheInner` when using the module?

Reply:
okay, cool. Then I wait for that change before merging this PR into w3.

Reply:
Done.